client/packages/common/src/ui/components/domain/Dashboard/Widget.tsx

```diff
@@ -29,9 +32,21 @@ export const Widget: React.FC<PropsWithChildren<WidgetProps>> = ({
       boxShadow: theme => theme.shadows[2],
     }}
   >
-    <Typography sx={{ fontSize: '14px', fontWeight: 'bold' }}>
-      {title}
-    </Typography>
+    <Grid container alignItems="center">
+      {Icon && (
```

Comment:
Can change later... not a fan of the background colour on icon but easy to change (: